src/test/java/org/ambraproject/rhino/service/IngestionTest.java

```diff
@@ -80,6 +81,7 @@
  * <p/>
  * Better, less hackish tests are desirable in the unit test classes that explicitly test particular services.
  */
+@Ignore
```

Comment:
This whole file has nothing but commented tests. junit complains when a test class contains no tests, so this `@Ignore` is there to satisfy that.

Reply:
Got it.  I was curious because of this comment in a file you deleted:
> TODO: make BaseRhinoTest transactional, and remove this.  IngestionTest currently breaks when this is done.

I tried uncommenting these tests and immediately retreated - there is more cruft there than just what that comment implied.  They are not passable.  I'm not prepared to delete them because the may still serve as a delcaration of intent.

Reply:
Yeah, commented tests always scare me. Who knows when they last passed?